### PR TITLE
Update storageos-operator to version 0.3.0

### PR DESCRIPTION
**Pull request by darkowlzz**

### CODE REVIEW

**Comment by croomes** [comment]
Hi @PennyScissors - we (StorageOS) have reviewed & tested this update.  We also have https://github.com/rancher/helm3-charts/pull/19 open for a Helm 3 version.

Please let us know if you have questions.  Thanks!

File: proposed/storageos-operator/v0.3.0/Chart.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+version: 0.3.0
 tillerVersion: ">=2.10.0"
 keywords:
 - storage
 - block-storage
 - volume
 - operator
 home: https://storageos.com
-icon: https://storageos.com/wp-content/themes/storageOS/images/logo.svg
+icon: https://docs.storageos.com/images/logo-helm.svg
```
Comment (PennyScissors) [inline proposed/storageos-operator/v0.3.0/Chart.yaml:13]: Please make a local copy of the logo in the parent folder `storageos-operator/` and reference the image here as `file://../logo-helm.svg`. This way the logo can be displayed in airtight environments.

File: proposed/storageos-operator/v0.3.0/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name: storageos-operator
+version: 0.3.0
+tillerVersion: ">=2.10.0"
+keywords:
+- storage
+- block-storage
+- volume
+- operator
+home: https://storageos.com
+icon: https://docs.storageos.com/images/logo-helm.svg
```
Comment (PennyScissors) [inline proposed/storageos-operator/v0.3.0/Chart.yaml:13]: Please make a local copy of the logo in the parent folder `storageos-operator/` and reference the image here as `file://../logo-helm.svg`. This way the logo can be displayed in airtight environments.

Comment (darkowlzz) [reply proposed/storageos-operator/v0.3.0/Chart.yaml]: Thanks. Added.

**Pull request merged**